frontend/src/modules/activity/config/filters/channel/config.ts

```diff
@@ -13,11 +17,11 @@ const channel: CustomFilterConfig = {
   component: ChannelFilter,
   options: {
   },
-  queryUrlParser: queryUrlParserByType[FilterConfigType.SELECT],
-  itemLabelRenderer(value: SelectFilterValue, options: SelectFilterOptions, data: any): string {
-    return itemLabelRendererByType[FilterConfigType.SELECT]('Channel', value, data);
+  queryUrlParser: queryUrlParserByType[FilterConfigType.MULTISELECT],
+  itemLabelRenderer(value: MultiSelectFilterValue, options: MultiSelectFilterOptions, data: any): string {
+    return itemLabelRendererByType[FilterConfigType.MULTISELECT]('Channel', value, data);
   },
-  apiFilterRenderer({ value, include }:SelectFilterValue): any[] {
+  apiFilterRenderer({ value, include }:MultiSelectFilterValue): any[] {
```

Comment:
The filter is being applied with an "and" operator. However, there will never be multiple channels in one activity. Can you apply a similar logic that you did with type in organizations, where you use the or operator?